lib/Cleantalk/ApbctWP/State.php

```diff
@@ -294,7 +294,9 @@ class State
             'last_send_time'          => 0,
             'last_send_amount'        => 0,
             'last_update_time'        => 0,
+            'last_update_way'         => '',
```

Comment:
Unused flag

Reply:
The flag is using here https://github.com/CleanTalk/wordpress-antispam/blob/sfw_shutdown/cleantalk.php#L1285